platformio.ini

```diff
@@ -33,6 +33,7 @@ platform = espressif32
 board = esp32dev
 framework = ${env.framework}
 upload_speed = ${env.upload_speed}
+;upload_speed = 926100
```

Comment:
this speed fails with your board?

Reply:
In the ESP32 devkit 1500000 not work.
But 926100 not work in the rest.